lib/permutations.rb

```diff
@@ -1,4 +1,28 @@
 
 def permutations?(string1, string2)
+  hash = {}
+  string1 = string1.split('')
+  string2 = string2.split('')
```

Comment:
Ruby does have a `.each_char` method so you don't technically need to split the string.